dm_api_account/apis/account_api.py

```diff
@@ -0,0 +1,66 @@
+import requests
+
+
+class AccountApi:
+
+    def __init__(
+            self,
+            host,
+            headers=None
+    ):
+        self.host = host
+        self.headers = headers
+
+    def post_v1_account(
+            self,
+            json_data
+    ):
+        """
+        Register new user
+        :param json_data:
+        :return:
+        """
+        response = requests.post(
+            url=f'{self.host}/v1/account',
+            json=json_data
+        )
+        return response
+
+    def put_v1_account_token(
+            self,
+            token
+    ):
```

Comment:
Лучше добавить, чтобы можно было пробрасывать параметры requests
```suggestion
    **kwargs
    ):
```